### PR TITLE
Feature/icap mme

### PR DESCRIPTION
**Pull request by bbakernoaa**
This adds on top of #132 adding the ICAP_MME to monetio.  ICAP MME is the 9 member multi member ensemble system developed at NRL. It is available through https://usgodae.org/metools/ensemble/

### CODE REVIEW

File: monetio/models/icap_mme.py [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+        print("\n File Exists: " + fname)
+
+
+def open_dataset(date, product="MMC", data_var="modeaod550"):
+    """
+    Parameters
+    ----------
+    date : str or datetime-like
+        The date for which to open the dataset.
+        2022-10-29 to current is available.
```
Comment (zmoon) [inline monetio/models/icap_mme.py]: Looks like the first non-MME is 2022-09-01
and the first MME is 2014-11-28

Comment (zmoon) [reply monetio/models/icap_mme.py]: Also we should provide a description of what MME, C4, MMC mean

Comment (bbakernoaa) [reply monetio/models/icap_mme.py]: MME is the same thing as MMC but for whatever reason the changed the name from MultiModelEnsemble to MultiModelConcensus

Comment (bbakernoaa) [reply monetio/models/icap_mme.py]: C4 is the Consensus from the 4 models with DA

File: monetio/models/icap_mme.py [hunk 2]
```diff
@@ -0,0 +1,189 @@
+import pandas as pd
+
+server = "ftp.star.nesdis.noaa.gov"
+base_dir = "/pub/smcd/VIIRS_Aerosol/npp.viirs.aerosol.data/epsaot550/"
+
```
Comment (zmoon) [inline monetio/models/icap_mme.py]:
```suggestion
```
Don't seem to be used here.

File: monetio/models/icap_mme.py [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+        raise ValueError(f"Invalid input for 'data_var': Valid values are {valid_data_vars}.")
+
+    urls, fnames = build_urls(d, filetype=product, data_var=data_var)
+    url = urls.values[0]
+    fname = fnames.values[0]
+    print(url)
+    print(fname)
+    if check_remote_file_exists(url) is False:
+        raise ValueError(f"File does not exist on ICAP HTTPS server: {url}")
+    retrieve(url, fname)
```
Comment (zmoon) [inline monetio/models/icap_mme.py]: Could make downloading to disk optional. `xr.open_dataset(io.BytesIO(r.content), engine="h5netcdf")` seems to work for loading the dataset from memory without saving to disk first.

File: monetio/models/icap_mme.py (hunk 1 above)
Comment (zmoon) [reply monetio/models/icap_mme.py]: Thanks. [Seems like](https://usgodae.org/ftp/outgoing/nrl/ICAP-MME/2022/202209/) 2022-09-11 is the last MME. 2022-09-01 -- 11 have both MME and MMC.

File: monetio/models/icap_mme.py (hunk 3 above)
Comment (bbakernoaa) [reply monetio/models/icap_mme.py]: oh interesting... should we add this to the nesdis_eps_viirs_nrt.py too?

File: monetio/models/icap_mme.py (hunk 1 above)
Comment (zmoon) [reply monetio/models/icap_mme.py]: I added the date info to the docstrings.

**Comment by bbakernoaa** [comment]
@zmoon this works great on my end

File: monetio/models/icap_mme.py [hunk 4]
```diff
@@ -0,0 +1,276 @@
+"""
+ICAP-MME
```
Comment (zmoon) [inline monetio/models/icap_mme.py:2]: @bbakernoaa does this all sound okay?

File: monetio/models/icap_mme.py [hunk 5]
```diff
@@ -0,0 +1,276 @@
...
+facing the operational aerosol community."
+
+The ICAP multi-model ensemble (ICAP-MME) is constructed from
+the following aerosol forecast systems:
+
+- ECMWF CAMS
+- JMA MASINGAR
+- NASA GEOS-5
+- NRL NAAPS
+- NOAA NGAC
```
Comment (bbakernoaa) [inline monetio/models/icap_mme.py]: This was NGAC but is now GEFS-Aerosols

Comment (bbakernoaa) [reply monetio/models/icap_mme.py]: There should be 9 models

File: monetio/models/icap_mme.py (hunk 4 above)
Comment (bbakernoaa) [reply monetio/models/icap_mme.py:2]: I think Peng covered this

**Review by zmoon — APPROVED** [review]
@bbakernoaa I'm pretty happy with this unless you have more test ideas or something

**Pull request merged**